invenio_rdm_records/services/components/verified.py

```diff
@@ -9,15 +9,31 @@
 from flask import current_app
 from invenio_access.permissions import system_identity
 from invenio_drafts_resources.services.records.components import ServiceComponent
+from invenio_records_resources.services.uow import TaskOp
 from invenio_requests.tasks import request_moderation
 
 
-class ContentModerationComponent(ServiceComponent):
-    """Service component for content moderation."""
+class BaseHandler:
+    """Base class for content moderation handlers."""
+
+    def __init__(self, uow):
```

Comment:
minor: uow should have a default value `= None`. python3.12 is complaining that in the config the parameter is not given

Reply:
In the end I refactored this to pass the `uow` as part of each handler (see after I push)